react/SearchTitle.js

```diff
@@ -1,5 +1,6 @@
 import classNames from 'classnames'
 import React, { useContext, useMemo } from 'react'
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Is this still needed?

Reply:
I think so, since the eslint throws an error due to the use of `ramda` 😕